rpi5.yaml

```diff
@@ -202,3 +201,26 @@ parameters:
           MACHINE: "raspberrypi5"
           SOC_FAMILY: "bcm2712"
           ZEPHYR_DOM0_DIR: "zephyr"
+
+  DOMD_ROOT:
+    desc: "Domd root device"
+    mmc:
+      default: true
+      overrides:
+        variables:
+          XT_DOMD_DTB_NAME: "%{SOC_FAMILY}-%{MACHINE}-mmc.dtb"
+        components:
+          domd:
+            builder:
+              conf:
+                - [MACHINE_FEATURES:remove, "domdusb"]
```

Comment:
You don't need to remove what is not present

Reply:
Ok